src/Fixture/ApiConfigurationFixture.php

```diff
@@ -66,7 +69,7 @@ protected function configureOptionsNode(ArrayNodeDefinition $optionsNode): void
                 ->scalarNode('api_client_id')->end()
                 ->scalarNode('api_client_secret')->end()
                 ->integerNode('pagination_size')->defaultValue(100)->end()
-                ->booleanNode('is_enterprise')->defaultFalse()->end()
+                ->booleanNode('edition')->defaultValue('ee')->end()
```

Comment:
don't forget to mention that change in your changelog

Reply:
done in UPGRADE.MD